public/css/module.less

```diff
@@ -6,6 +6,18 @@
 @cert-segment-color-3: #1982C4;
 @cert-segment-color-4: #6A4C93;
 
+.empty-state {
```

Comment:
Is this still necessary?

Reply:
Yes! The empty state default styles apply only when it's within an `item-list/list-item` classes.

Reply:
Ok, why? I don't think that restriction makes sense.

Reply:
The empty state class is also used in the host/service object detail view, for example, and there it is styled differently than when it is within an item-list. Thus, the emty-state is styled differently depending on where it is used, so we can't apply the `item-list` styles globally.

Reply:
Sure, but I meant that you are basically copying the styles since the views here are list/main views but use tables instead of being item lists. I can't reuse the styles for the empty state (for lists/main views) if I want to, but that should be possible.

Reply:
There's `EmptyStateBar` now for the style which can be used on its own without a item list or item table. The margin however is related to the placement, this must be kept here but should be applied only to the tables where required.

Reply:
> There's `EmptyStateBar` now for the style which can be used on its own without a item list or item table. The margin however is related to the placement, this must be kept here but should be applied only to the tables where required.

Nice!